docs/tutorial/overview.md

```diff
@@ -0,0 +1,5 @@
+# Overview
+
+The wordpress-k8s charm helps deploy a horizontally scalable WordPress application with ease and
+also helps operate the charm by liaising with the Canonical Observability Stack(COS). This tutorial
+will walk you through each step of deployment to get a basic WordPress application working.
```

Comment:
Maybe change the end of this to `a WordPress deployment`. The basic qualifier might indicate that the charm be default is basic, I'm guessing it is to indicate that more can be done with the charm. Perhaps that can be left to the end of the tutorial? Including `working` could imply that the charm is buggy and needs a tutorial to resolve those bugs - changing it to refer to a deployment carriers the intent that the tutorial will get you up and running with Wordpress

Reply:
Thank you for the clear explanation, makes sense. Changed the wording from `application working` to `WordPress deployment`.
```
This
tutorial will walk you through each step of deployment to get a basic WordPress deployment.
```
Thank you for the suggestion!